src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,23 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1030004
+name: Mollaret’s meningitis
+def: "Mollaret’s meningitis, also known as recurrent benign lymphocytic meningitis, is a rare clinical disorder characterized by at least three recurrent episodes of meningitis associated with spontaneous recovery with or without antiviral therapy. Herpes simplex virus type 2 has frequently been implicated in the setting of this illness. RBLM is rare with prevalence rates estimated at 2 per 100,000 persons." [https://orcid.org/0000-0001-9310-0163]
```

Comment:
the PMID x-ref should be here for the definition.

Reply:
We avoid repeating the name of the disease at the beginning of definitions. Therefore the definition should be:
"A rare clinical disorder characterized by (...)" 
You can add "recurrent benign lymphocytic meningitis" as a synonym (which you have already done) instead of adding in the definition.

Reply:
fixed